support.md

```diff
@@ -0,0 +1,26 @@
+# Support
+
+## Microsoft Support Policy
+
+If issues are encountered when using template and code in this repo, users will be able to engage Microsoft support via their usual channels. Please provide corelation IDs where possible when contacting support to be able to investigate issue effectively and in timely fashion. For instruction on how to get deployments and correlation ID, please follow this link [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-history?tabs=azure-portal#get-deployments-and-correlation-id).
+
+Following list of issues are within the scope of Microsoft support:
+
+- Tenant, Management Group, Subscription and Resource Group deployment
+- Az PowerShell commandlets e.g. Get-AzManagementGroup
+- ARM Deployment Issues e.g. template validation, CheckAccess API etc.
+- Git Actions/Azure DevOps pipeline itself however please note that PowerShell module will be supported via community support only.
+
+Any issues that deemed outside of the above list by Microsoft support and/or requires bugfix in the Template or Code in the repo, Microsoft support will redirect user to file the issue on GitHub.
+
+Project maintainers and community aim to get issues resolved in timely fashion as per community support policy of this repo.
+
+## Community Support Policy
+
+Project maintainers will aim to respond within 3 business days to get a meaningful response for any new issues.
```

Comment:
I think the only challenge here is if I am a large enterprise looking to take a dependency on AzOps, this support policy does not inspire a lot of confidence that my issue will get resolved quickly.